### PR TITLE
Uses editorconfig to maintain consistent coding styles

### PR DESCRIPTION
**Pull request by ManasJayanth**
Fix for #6626

### CODE REVIEW

**Comment by yurydelendik** [comment]
My stats show more than 130 files shall be affected, you diff has only 42.

stripCommentHeaders was not modified too.

**Comment by timvandermeij** [comment]
`stripCommentHeaders` from https://github.com/mozilla/pdf.js/blob/d8ddde26b7560b9ee6475bb664dd8d0b26e5b980/make.js#L606-L615 should also be removed, along with the calls to that function. As Yury mentioned, probably there are more files with the header, so use a search to find out which ones exactly.

**Comment by yurydelendik** [comment]
> stripCommentHeaders ... should also be removed

Actually not remove, but modifed to fit new header format.

**Comment by timvandermeij** [comment]
Right, that should be done. It seems that the files in `/web` are missing in this PR (example: https://github.com/mozilla/pdf.js/blob/master/web/annotations_layer_builder.js). They also have this header.

**Comment by prometheansacrifice** [comment]
@yurydelendik Was it 131 files? 4 of them are from `build` directory.

**Comment by yurydelendik** [comment]
> Was it 131 files

I think it was 132

File: web/chromecom.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
-/* -*- Mode: Java; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* Copyright 2013 Mozilla Foundation
+/* Copyright 2014 Mozilla Foundation
```
Comment (yurydelendik) [inline web/chromecom.js]: Can you revert changes of the year of copyright? It's hard to get it right and out of scope of this PR

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+insert_final_newline = true
+max_line_length = 80
+trim_trailing_whitespace = true
+
+[*.md]
+max_line_length = 0
+trim_trailing_whitespace = false
+
+[COMMIT_EDITMSG]
+max_line_length = 0
```
Comment (Snuffleupagus) [inline .editorconfig]: Nit: missing newline at the end of this file.

File: web/chromecom.js (hunk 1 above)
Comment (prometheansacrifice) [inline web/chromecom.js]: Sorry about this Reverting

**Comment by Snuffleupagus** [comment]
It appears that this PR is still missing most of the files in https://github.com/mozilla/pdf.js/tree/master/extensions/firefox/content/.

File: external/webL10n/l10n.js [hunk 3]
```diff
@@ -1,5 +1,4 @@
-/**
- * Copyright (c) 2011-2013 Fabien Cazenave, Mozilla.
+/* Copyright 2014 Mozilla Foundation
```
Comment (Snuffleupagus) [inline external/webL10n/l10n.js]: Please revert this change.

Comment (Snuffleupagus) [inline external/webL10n/l10n.js]: Please revert this change; and also the rest of the changes of Copyright year in the rest of this patch.

File: make.js [hunk 4]
```diff
@@ -607,7 +605,6 @@ function stripCommentHeaders(content, filename) {
   // Strip out all the vim/license headers.
   var notEndOfComment = '(?:[^*]|\\*(?!/))+';
   var reg = new RegExp(
-    '\n/\\* -\\*- Mode' + notEndOfComment + '\\*/\\s*' +
```
Comment (yurydelendik) [inline make.js]: add '\n' to the next line, we want to not affect very first comment

also can you fix the comment above -- it does not reflect the purpose of the code below

**Comment by yurydelendik** [comment]
Hmm, grab-to-pan.js header is gone from viewer.js. @Rob--W is it okay or you prefer to keep it? We probably we just need move copyright into main file header.

**Comment by Rob--W** [comment]
@yurydelendik I don't mind either way.

**Comment by yurydelendik** [comment]
Added comment about that at #3775 (maybe @prometheansacrifice can look into that later :)

Thank you for the patch.
